orttraining/orttraining/python/ort_trainer.py

```diff
@@ -598,9 +600,12 @@ def __init__(self, model, loss_fn, model_desc, training_optimizer_name, map_opti
         self.enable_grad_norm_clip_ = enable_grad_norm_clip
         self.frozen_weights_ = frozen_weights
         self.opset_version_ = _opset_version
-        self.loss_scale_input_name = ''
         self.state_dict_ = None
 
+        # use this special string to workaround a corner case that external loss_scale is passed into train_step as kargs.
+        # see prepare_input_and_fetches for more details.
+        self.loss_scale_input_name = 'unset_loss_scale_input_name'
```

Comment:
maybe change name to 'default'?
```suggestion
        self.loss_scale_input_name = 'default_loss_scale_input_name'
```